mitodl/ocw-studio
# import for learning_resource_types

#### Pre-Flight checklist

#### What are the relevant tickets?
https://github.com/mitodl/ocw-studio/issues/948

#### What's this PR do?
This PR updates the `import_ocw_course_sites` management command to set the `learning_resource_types` metadata field. It also creates a new `overwrite_ocw_course_content` management command  to overwrite individual fields in WebsiteContent or WebsiteContent metadata from plone data without overwriting any other changes made  in ocw-studio.

In the future it makes sense to update the `overwrite_ocw_course_content` management command  to overwrite other types of Website content as well

#### How should this be manually tested?
Run
```
docker-compose run web ./manage.py import_ocw_course_sites --bucket ocw-to-hugo-output-qa --filter 6-007-electromagnetic-energy-from-motors-to-lasers-spring-2011
```

Verify that resource records for  `6-007-electromagnetic-energy-from-motors-to-lasers-spring-2011` have `learning_resource_types` populated in ui

Modify the titles, descriptions and any other fields of some pages or resources for the course.

Run
```
docker-compose run web ./manage.py overwrite_ocw_course_content --bucket ocw-to-hugo-output-qa --filter 6-007-electromagnetic-energy-from-motors-to-lasers-spring-2011 --content-field title
```
Verify that the titles are returned to the original values but all other field values remain unchanged

Run
```
docker-compose run web ./manage.py overwrite_ocw_course_content --bucket ocw-to-hugo-output-qa --filter 6-007-electromagnetic-energy-from-motors-to-lasers-spring-2011 --content-field metadata.description
```
Verify that the descriptions are returned to the original values but all other field values remain unchanged
